jax_unirep/utils.py

```diff
@@ -230,13 +196,34 @@ def validate_mLSTM_params(params: Dict, n_outputs):
 
 
 def load_params(folderpath: Optional[str] = None):
-    """Load params for passing to evotuning stax model."""
-    return (
-        load_mlstm_params(folderpath=folderpath),
-        (),
-        load_dense_params(folderpath=folderpath),
-        (),
-    )
+    """
+    Load params for passing to evotuning stax model.
+
+    The weights are saved as a single pickle file.
+    We did this in version 1.1 to unify how weights are stored and dumped.
+    When loaded into memory, the weights object `params`
+    will be a nested tuple of arrays and dictionaries. In order, they are:
+
+    - embedding params
+    - mLSMT1900 params (with gating weights `g*`, matrix multiplication weights `w*`, and bias `b` as keys)
+    - etc.
```

Comment:
```suggestion
    - dense params to predict one-hot encoded next letter.
```

Reply:
lol sorry, got lazy in copying your docstring